market_api/schema.py

```diff
@@ -0,0 +1,25 @@
+
+from marshmallow import Schema, fields
+
+
```

Comment:
Please add an input schema as well, and use it to validate the `vin` input is a valid VIN number.  The `vin` input is from the function above 
`de get_vin_history(self, vin):`
Reference:  https://stackoverflow.com/questions/30314850/vin-validation-regex